partials/forms/radio-group.html

```diff
@@ -1,7 +1,7 @@
 <div id="{{key}}-group" class="form-group{{#display}} {{display}}{{/display}}{{#error}} validation-error{{/error}}">
     {{#options}}
         <label class="block-label" for="{{key}}-{{value}}">
-            <input type="radio" name="{{key}}" id="{{key}}-{{value}}" value="{{value}}" aria-required="true"{{#selected}} checked="checked"{{/selected}}>
+            <input type="radio" name="{{key}}" id="{{key}}-{{value}}" value="{{value}}" aria-required="true"{{#toggle}} data-toggle="{{toggle}}"{{/toggle}}{{#selected}} checked="checked"{{/selected}}>
```

Comment:
These attributes are getting to the point where we may wish to consider adding some newlines.